.gitignore

```diff
@@ -1,2 +1,3 @@
 composer.lock
 vendor/
+.idea/
```

Comment:
Please, remove IDE files exceptions. This should be in your global ignore configuration.

Reply:
Fixed.